src/GameplayKit/GKGridGraph.cs

```diff
@@ -0,0 +1,29 @@
+﻿//
+// GKGridGraph.cs: Implements some nicer methods for GKGridGraph
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+#if XAMCORE_2_0 || !MONOMAC
+using System;
+using XamCore.ObjCRuntime;
+using Vector2i = global::OpenTK.Vector2i;
+
+namespace XamCore.GameplayKit {
+	public partial class GKGridGraph {
+		
+		public GKGridGraphNode GetNodeAt (Vector2i position)
```

Comment:
The original generated method was `virtual`, so this is a breaking change (missing `virtual`).